lib/workers/repository/update/pr/index.ts

```diff
@@ -57,6 +63,11 @@ export type ResultWithoutPr = {
 
 export type EnsurePrResult = ResultWithPr | ResultWithoutPr;
 
+interface PrRenVer {
+  prCreationVer: string;
+  prUpdateVer: string;
+}
```

Comment:
Please, use `PrBodyStruct` instead

Reply:
Also, check out `getPrBodyStruct()` function that populates `Pr::bodyStruct` field

Reply:
i'm not sure what you mean, 
do you want me to go to `Platform\Types`
then add an object called `renovatePrData` into this

```
export interface PrBodyStruct {
  hash: string;
  rebaseRequested?: boolean;
  renovatePrData: RenovatePrData
}
```

please confirm or elaborate what

Reply:
Yes, it's correct

Reply:
i added it but i'm not filling it anywhere, because i need it as a comment, in the prBody.
is there a reason to fill the object anywhere? like 
```
On creation:
pr.bodyStruct.renovatePrData.prUpdateVer = pkg.version 
pr.bodyStruct.renovatePrData.prCreationVer = pkg.version

on update:
existingPr.bodyStruct.renovatePrData.prUpdateVer = pkg.version
```

Reply:
@zharinov  Thanks a lot for the amazing suggestions, im working on it